ray-operator/controllers/ray/common/pod.go

```diff
@@ -415,14 +415,12 @@ func BuildAutoscalerContainer(autoscalerImage string) corev1.Container {
 			},
 		},
 		Command: []string{
-			"ray",
```

Comment:
You'll need to update the `TestBuildPod_WithAutoscalerEnabled`  test in pod_test.go

Reply:
hi @andrewsykim , I have update it. Please check.